### PR TITLE
feat:added cart

### PR DESCRIPTION
**Pull request by ianuragsharma**
image

### CODE REVIEW

**Review by Aditi0505 — APPROVED** [review]
LGTM

File: pages/cart.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    <div class="cart-btn flex-column">
+                        <button class="btn btn-outlined-secondary ">Remove From Cart</button>
+                        <button class="btn btn-solid-primary  text-white">Move to Wishlist</button>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div class="price-container">
+            <h4 class="fw-500 ">Price Details</h4>
```
Comment (anupkgurung) [inline pages/cart.html:92]: what is fw-?

**Review by aurobindoGupta — APPROVED** [review]
nice

**Review by anupkgurung — APPROVED** [review]
if fw is not something widely used class name convention than please try to use some meaning full class name. Rest looks good.

File: pages/cart.html (hunk 1 above)
Comment (ianuragsharma) [reply pages/cart.html:92]: its font-weight, i thought naming it something like font-weight-500 initially, but it was very lengthy,so i settled with fw-500.

**Pull request merged**